src/elements.jl

```diff
@@ -145,7 +144,12 @@ end
 function local_potential_fourier(el::ElementCohenBergstresser, q::T) where {T <: Real}
     q == 0 && return zero(T)  # Compensating charge background
 
+    # Number of digits to keep in rounding (depending on element type)
+    digits = 10
+    eps(T) > 1e-13 && (digits = 3)
+    eps(T) > 1e-6 && (digits = 2)
+
```

Comment:
That logic looks very ad hoc. Why not just round? The argument is an integer up to machine precision, no? There could perhaps be trickiness for very large q, but we don't care about those anyway.

Reply:
Not if you use a different lattice constant in the lattice and in the CohenBergstresser potential, which you should not, but I definitely want it to flag somehow.

Reply:
I see. But this is conflating two different things that are unrelated: numerical precision and whether you are doing something weird with the lattice or not. I'd just use `digits=2` always, with a comment saying that this only allows for small deformations of the unit cell. A more clever option is to cook up an interpolation formula that passes through the three data points and gives V(q) for all q, but we don't particularly care anyway...